samples/snippets/update_realm.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Game Servers sample for updating a realm.
+
+Example usage:
+    python update_realm.py --project-id <project-id> --location <location> --realm-id <realm-id>
+"""
+
+import argparse
+
+from google.cloud import gaming
+from google.cloud.gaming_v1.types import realms
+from google.protobuf import field_mask_pb2 as field_mask
+
+
+# [START cloud_game_servers_update_realm]
+def update_realm(project_id, location, realm_id):
```

Comment:
Do we need update samples for realms?  For go we only have update rollout samples.  We have no place for this in the documentation.

Reply:
Labels are important for GCP services. I was testing label update with the client library. Then I kept it here as it might be useful users.

I can remove them if 1-1 mapping between doc and sample is required.